.editorconfig

```diff
@@ -0,0 +1,3 @@
+[*.{js,ts,tsx,jsx,json}]
+indent_style = space
+indent_size = 2
```

Comment:
needs new line after this...

Reply:
is there any setting which can add new line automatically on save ??
(ex: "files.insertFinalNewline": true in vscode config file)

Reply:
done